.travis.sh

```diff
@@ -60,19 +60,52 @@ tests_sequence() {
     )
 }
 
+tests_sequence_ios_sim() {
```

Comment:
It seems like this should have `aarch64` or `arm64` in the name, and `test_sequence` should have `x86_64` in the name.

Reply:
Yeah, I don't love it. The `test_sequence` function is used for android, qemu and the iOS targets. I think `tests_sequence_aarch64_ios_sim` might be okay but it's pretty un-DRY.

Reply:
Ah, I see. I agree that's a better name.